cmake/app/FindPythonInterp.cmake

```diff
@@ -0,0 +1,167 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
Should this file be in `cmake/app`? Maybe in `cmake/backports/`?

Reply:
Done.